src/transformers/utils/doc.py

```diff
@@ -1118,6 +1131,11 @@ def docstring_decorator(fn):
         code_sample = filter_outputs_from_example(
             code_sample, expected_output=expected_output, expected_loss=expected_loss
         )
+        if real_checkpoint is not None:
+            intro = FAKE_MODEL_DISCLAIMER.format(
```

Comment:
Is it possible to not format `here`, but just simply `code_sample = intro + code_sample` directly, and let `built_doc = code_sample.format(**doc_kwargs)` at the end to handle the format just once?